app/src/main/java/org/oppia/android/app/home/HomeViewModel.kt

```diff
@@ -149,6 +190,22 @@ class HomeViewModel(
       }
   }
 
+  /**
+   * Returns a list of [HomeItemViewModel]s corresponding to [ComingSoonTopicListViewModel]  all the upcoming topics available in future and to be
+   * displayed for this profile (see [ComingSoonTopicsViewModel]), or an empty list if the profile does not have any
+   * ongoing stories at all.
+   */
+  private fun computeComingSoonTopicViewModelList(
+    comingSoonTopicList: ComingSoonTopicList
+  ): List<ComingSoonTopicsViewModel> {
+    return comingSoonTopicList.upcomingTopicList.mapIndexed { topicIndex, topicSummary ->
```

Comment:
what's the use of topicIndex here or why have you used mapIndexed here?

Reply:
Topic index is not used. Removed it.